website/addons/github/model.py

```diff
@@ -309,14 +309,6 @@ def before_page_load(self, node, user):
                     ' Users can view the contents of this private GitHub '
                     'repository through this public project.'
                 )
-            else:
```

Comment:
With this change, alert messages are assumed to be HTML unsafe. Because this section includes a link, the `user` and `repo` properties would need to be sanitized in order to safely turn that off.

I basically looked at the logic here, and found that this message is only displayed if the repo's permissions and the node's permissions differ. That seemed like a bug to me, as the most common use case is likely going to be a public project with a public repo - but in any case, the message would be applicable regardless of the permissions on either object.

Given the choice between figuring out the intended behavior and removing what seemed to me an inconsistent message, I removed it.

As far as I'm concerned, it's your call if you want me to add it back in. I'd be happy to do so.